eureka/S3_data_reduction/miri.py

```diff
@@ -68,7 +68,16 @@ def read(filename, data, meta):
         if meta.firstFile:
             print('  WARNING: The timestamps for the simulated MIRI data are currently hardcoded '
                   'because they are not in the .fits files themselves')
-        if 'new_drift' in data.filename:
+
+        if 'WASP_80b' in data.filename and 'transit' in data.filename:
```

Comment:
Why WASP-80b?  The simulated data are for NGTS-10b.  Is there another dataset out there?

Reply:
I had a custom WASP-80b simulation made as it's one of Tom's GTO targets and we wanted to practice completing a generate spectrum -> simulate data -> fit simulations -> retrieve on fit circle to see how well we can do as a team.